jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9VirtualMachine.java

```diff
@@ -555,6 +573,38 @@ public String startLocalManagementAgent() throws IOException {
 		return result;
 
 	}
+	
+	/**
+	 * Generate a text description of a target JVM's heap, including the number and
+	 * sizes of instances of each class.
+	 * 
+	 * @param opts String options: "-live" for live object only, or "-all" for all
+	 *             objects
+	 * @return byte stream containing the UTF-8 text of the formatted output
+	 */
+	public InputStream heapHisto(Object... opts) {
```

Comment:
Why is the signature not `String...`?

Reply:
This is for compatibility with the reference implementation:

`sun.tools.attach.HotSpotVirtualMachine#public InputStream heapHisto(Object... var1)`.